src/modules/core/link_timeremap.c

```diff
@@ -333,7 +336,10 @@ static int link_get_image_nearest( mlt_frame frame, uint8_t** image, mlt_image_f
 	if ( src_frame )
 	{
 		uint8_t* in_image;
-		if ( !mlt_frame_get_image( src_frame, &in_image, format, width, height, 0 ) )
+		mlt_service_lock( MLT_LINK_SERVICE(self) );
+		int result = mlt_frame_get_image( src_frame, &in_image, format, width, height, 0 );
+		mlt_service_unlock( MLT_LINK_SERVICE(self) );
+		if ( !result )
```

Comment:
Sorry for the nit, but could you make the "if" statements match? I do not have a preference, just choose "!result" or "result != 0" so they match.

Then we can get this merged in just in time for the next release!

Reply:
Done 👍